sky/cli.py

```diff
@@ -2752,9 +2759,40 @@ def show_gpus(
     To show all accelerators, including less common ones and their detailed
     information, use ``sky show-gpus --all``.
 
+    To refresh latest price info, use ``sky show-gpus --refresh``.
+
     NOTE: If region is not specified, the price displayed for each instance type
     is the lowest across all regions for both on-demand and spot instances.
     """
+
+    def fetch_latest_price(cloud_name: str):
+        if cloud_name not in ['aws', 'azure', 'gcp']:
+            click.echo(
+                f'Fetcher for cloud "{cloud_name}" is not supported now.',
+                err=True)
+            return
+        try:
+            cloud_fetcher_module = importlib.import_module(
+                f'sky.clouds.service_catalog.data_fetchers.fetch_{cloud_name}')
+        except ModuleNotFoundError:
+            raise ValueError(
+                'Cannot find module "sky.clouds.service_catalog'
+                f'.data_fetchers.fetch_{cloud_name}" for cloud "{cloud_name}".'
+            ) from None
+        try:
+            fetch = getattr(cloud_fetcher_module, 'fetch')
+        except AttributeError:
+            raise AttributeError(f'Module "fetch_{cloud_name}" does not '
+                                 f'implement the "fetch" method') from None
+        fetch()
+
+    if refresh:
+        if cloud is None:
+            for cl in clouds.CLOUD_REGISTRY.values():
+                fetch_latest_price(cl._REPR.lower())
+        else:
+            fetch_latest_price(cloud)
```

Comment:
This design can be pretty unclear. We may not want to have the user run the price fetcher as that will be quite time-consuming + error-prone. We are updating the price data for AWS mainly by periodically fetching from our hosted catalog. https://github.com/skypilot-org/skypilot/blob/3635b0dcb2a2a9653ee7acf5b52b4863c6caf1c1/sky/clouds/service_catalog/aws_catalog.py#L47-L48

We may want to do something similar for the GCP and Azure's catalog as well, i.e. try to add a github CI to fetch the latest catalog periodically https://github.com/skypilot-org/skypilot-catalog/blob/master/.github/workflows/update-aws-catalog.yml